app/jenkins-for-jira-ui/src/components/InProductHelpDrawer/InProductHelp.styles.tsx

```diff
@@ -1,22 +1,38 @@
 import { css } from '@emotion/css';
 import { token } from '@atlaskit/tokens';
 
+export const iphActionContainer = css`
+	margin-left: 16px;
+`;
+
 export const inProductHelpActionLink = css`
 	background-color: inherit;
 	border: none;
 	color: ${token('color.link')};
+	cursor: pointer;
 	margin-bottom: ${token('space.025')};
-	padding: ${token('space.0')} !important;
+	padding: ${token('space.0')};
+`;
 
-	span {
-    font-weight: normal;
-    margin: ${token('space.0')};
-		line-height: 20px;
-		text-align: left;
-		white-space: normal !important;
-  }
+export const inProductHelpActionButton = css`
```

Comment:
When you check out the component for this you'll see I'm no longer using the atlaskit Button component. Reason: there's no way to align a button element so it runs on from a paragraph element like so

<img width="512" alt="Screenshot 2024-01-10 at 5 33 19 pm" src="https://github.com/atlassian/jenkins-for-jira/assets/37155488/1ebc46c1-18c1-45a5-ab82-21a30ab01503">

Had to do a lot of fuckery to get it looking like the designs :laughcry: